### PR TITLE
join: add -t example

### PR DESCRIPTION
**Pull request by stsourlidakis**
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/join.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 `join {{file1}} {{file2}}`
 
+- Join two files using a comma (instead of space) as a field separator:
+
+`join -t ',' {{file1}} {{file2}}`
+
 - Join field3 of file1 with field1 of file2:
 
 `join -1 3 -2 1 {{file1}} {{file2}}`
```
Comment (agnivade) [inline pages/common/join.md]: While we are here, let's put 3 and 1 within tokens (`{{}}`). And same for 1 in the next example

Comment (sbrl) [reply pages/common/join.md]: and the `','` on line 11 too?

Comment (agnivade) [reply pages/common/join.md]: Yeah, good call.

**Comment by stsourlidakis** [comment]
@sbrl Seems better now, thanks!

**Pull request merged**